NotadogApi/Domain/Game/Room.cs

```diff
@@ -24,16 +24,19 @@ public class Room
     {
         public Guid Guid { get; }
         public List<User> Players { get; }
+        public List<User> MakedMovePlayers { get; }
```

Comment:
Think about thread safety. For example, what would happen if last two players press Not A Dog Button at the same time. Investigate this example https://refactoring.guru/ru/design-patterns/singleton/csharp/example#example-1 to solve the problem.

Reply:
it's a good point. Will do